R/extract.R

```diff
@@ -1842,21 +2037,92 @@ extract.glm <- function(model, include.aic = TRUE, include.bic = TRUE,
   )
   return(tr)
 }
-
+#' \code{\link{extract}} method for \code{glm} objects
+#'
+#' \code{\link{extract}} method for \code{glm} objects. These objects are
+#' created by the \code{\link[stats]{glm}} function in the \pkg{stats} package.
+#'
+#' @param model A model object.
+#' @param include.aic Report AIC in the GOF block?
+#' @param include.bic Report BIC in the GOF block?
+#' @param include.loglik Report the log likelihood in the GOF block?
+#' @param include.deviance Report the deviance in the GOF block?
+#' @param include.nobs Report the number of observations in the GOF block?
+#' @param ... Additional arguments for the \code{\link[base]{summary}} function
+#'   for \code{glm} objects.
+#' @return A \linkS4class{texreg} object.
+#'
+#' @method extract glm
+#' @aliases extract.glm
+#' @family extract
+#' @seealso \link{extract}
+#' @author Philip Leifeld
+#'
+#' @importFrom stats nobs
```

Comment:
I think there are a few additional functions that need to be imported.

Reply:
in the glm family? 
I included every function that has `<- extract.glm` assigned. Or I am not understanding something?

Reply:
As far as I can remember, a few functions from the `stats` package were used within the extract method. Can you locate them and add `@importFrom` statements for them as well. It wasn't just `nobs`.